### ISSUE TITLE
Issue while running virtual box

### ISSUE THREAD

**Issue by jeevast** [open-learning-exchange/open-learning-exchange.github.io#2309]
#### This issue is raised for the step 1 - planet installation with vagrant.

In some cases, even after enabling virtualization at BIOS, we may land up with the error stating that virtualization is unsupported in this machine. Below screenshot provides the complete description of the problem and this problem most likely to occur in linux machine.

### Steps to reproduce the problem

N/A

### Screenshots

![vagrant 4](https://user-images.githubusercontent.com/14145400/58440264-e23bb100-80a6-11e9-9e79-4600ad7e5f40.png)

### Proposed solution

A solution for this I found myself is to disable "secure boot" in BIOS located under "Boot Options" or to create a personal RSA public/private key as mentioned in this [blog](https://stegard.net/2016/10/virtualbox-secure-boot-ubuntu-fail/). Later on I was able to run a virtual box and vagrant after disabling secure boot.However, it's mandatory to have secure boot enabled while installing new operating system. Any alternate solution or discussion on this issue is most welcomed.

**Comment by lmmrssa** [comment]
@jeevast it looks an interesting issue. You can make a note about issue of **Secure Boot** mentioning that it needs to be disabled if you see this issue.
Regarding blog if you add it on wiki then please make sure you do not highlight it as that is completed for aspiring interns to try out.

**Comment by Kid243** [comment]
If my understanding is correct, even after running the virtual box once without creating the RSA public/private key enabling back "secure boot" would still end up with the same issue?

**Comment by jeevast** [comment]
@Kid243 yes, it would still end up with the same issue at least with linux machines.
